app/decorators/content_item_decorator.rb

```diff
@@ -9,6 +9,10 @@ def last_updated
     end
   end
 
+  def feedex_link
+    helpers.link_to "View anonymous feedback on FeedEx", "https://support.publishing.service.gov.uk/anonymous_feedback?path=#{object.base_path}"
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/107/commits/9df386807fa20a3e063ddca92e32b0399a4fb3da